src/service/db/folders.rs

```diff
@@ -13,37 +13,42 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
The changes in this module swap out the logic that uses the `meta` table for logic that uses the new `folder` table.